File: Modul 3/Soal 4/PRAK304-2310817210005-GALIH_AJI_SABDARAYA.py
n = int(input())
if n == 0:
    print("\nNol")
elif n > 0 and n < 10:
    print("\nSatuan")
elif n >= 10 and n < 20:
    print("\nBelasan")
elif n >= 20 and n < 100:
    print("\nPuluhan")
elif n >= 100:
    print("\nAnda Menginput Melebihi Limit Bilangan")

File: Modul 3/Soal 4/PRAK304-2310817210005-GALIH_AJI_SABDARAYA.c
#include <stdio.h>

int main(){
    int a;
    scanf("%d", &a);
    if (a == 0){
        printf("\nNol");
    } else if (a > 0 && a < 10){
        printf("\nSatuan");
    } else if (a >= 10 && a < 20){
        printf("\nBelasan");
    } else if (a >= 20 && a < 100){
        printf("\nPuluhan");
    } else if (a >= 100){
        printf("\nAnda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}